terraform/modules/apps/frontend/main.tf

```diff
@@ -7,79 +7,66 @@ terraform {
-  cpu                      = 512
-  memory                   = 1024
-  task_role_arn            = var.task_role_arn
-  execution_role_arn       = var.execution_role_arn
-
-  proxy_configuration {
-    type           = "APPMESH"
-    container_name = "envoy"
-
-    properties = {
-      AppPorts         = "80"
-      EgressIgnoredIPs = "169.254.170.2,169.254.169.254"
-      IgnoredUID       = "1337"
-      ProxyEgressPort  = 15001
-      ProxyIngressPort = 15000
-    }
-  }
+data "aws_secretsmanager_secret" "secret_key_base" {
+  name = "frontend_app-SECRET_KEY_BASE"
 }
 
-resource "aws_ecs_task_definition" "console_definition" {
-  family                   = var.service_name
-  requires_compatibilities = ["FARGATE"]
-  container_definitions    = file("${path.module}/frontend_console.json")
-  network_mode             = "awsvpc"
-  cpu                      = 512
-  memory                   = 1024
-  task_role_arn            = var.task_role_arn
-  execution_role_arn       = var.execution_role_arn
+module "app" {
+  source                           = "../../app"
+  cpu                              = "512"
+  memory                           = "1024"
+  vpc_id                           = var.vpc_id
+  cluster_id                       = var.cluster_id
+  service_name                     = var.service_name
+  subnets                          = var.private_subnets
+  mesh_name                        = var.mesh_name
+  service_discovery_namespace_id   = var.service_discovery_namespace_id
+  service_discovery_namespace_name = var.service_discovery_namespace_name
+  task_role_arn                    = var.task_role_arn
+  execution_role_arn               = var.execution_role_arn
+  extra_security_groups            = [var.govuk_management_access_security_group]
+  container_definitions = [
+    {
 
-  proxy_configuration {
-    type           = "APPMESH"
-    container_name = "envoy"
-
-    properties = {
-      AppPorts         = "80"
-      EgressIgnoredIPs = "169.254.170.2,169.254.169.254"
-      IgnoredUID       = "1337"
-      ProxyEgressPort  = 15001
-      ProxyIngressPort = 15000
+      "name" : "frontend",
+      "image" : "govuk/frontend:fargate-statsd",
+      "essential" : true,
+      "environment" : [
+        { "name" : "RAILS_ENV", "value" : "production" },
+        { "name" : "ASSET_HOST", "value" : "www.gov.uk" },
+        { "name" : "GOVUK_APP_DOMAIN", "value" : "www.gov.uk" },
+        { "name" : "GOVUK_WEBSITE_ROOT", "value" : "www.gov.uk" },
+        { "name" : "WEBSITE_ROOT", "value" : "www.gov.uk" },
+        { "name" : "PLEK_SERVICE_CONTENT_STORE_URI", "value" : "https://www.gov.uk/api" },
+        { "name" : "PLEK_SERVICE_STATIC_URI", "value" : "https://assets.publishing.service.gov.uk" },
+        { "name" : "GOVUK_ASSET_ROOT", "value" : "https://assets.digital.cabinet-office.gov.uk" },
+        { "name" : "STATSD_PROTOCOL", "value" : "tcp" },
+        { "name" : "STATSD_HOST", "value" : "statsd.test.govuk-internal.digital" },
```

Comment:
As a follow-up you could consider taking some of these values from the vars which we're already passing down from `modules/govuk` (like govuk_app_domain and statsd_host for example). Don't worry about for this PR though - better to get this merged first.